code/script.js

```diff
@@ -1,9 +1,51 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
+
+
+//1 get the icon ids' and store as variables
+const minusElement1 = document.querySelector('#minus1')
+const plusElement1 = document.querySelector('#plus1')
+
+const minusElement2 = document.querySelector('#minus2')
+const plusElement2 = document.querySelector('#plus2')
+
+const minusElement3 = document.querySelector('#minus3')
+const plusElement3 = document.querySelector('#plus3')
```

Comment:
Nice! This could also be done by CSS styling, so you use the "active" class and then you can style with ::after or ::before and the CSS property `content` :)

Reply:
Ah nice tip, didn't think about this 🤩